### ISSUE TITLE
Add Google Drive integration test

### ISSUE THREAD

**Issue by dentarg** [dentarg/skuld#3]
Something like this:

``` diff
$ git diff
diff --git a/test/integration/skuld_test.rb b/test/integration/skuld_test.rb
index 06705bf..4ca6d97 100644
--- a/test/integration/skuld_test.rb
+++ b/test/integration/skuld_test.rb
@@ -19,4 +19,9 @@ class SkuldTest < Minitest::Test
     get "/test"
     assert last_response.ok?
   end
+
+  def test_demo
+    get "/1GQFvsrwWDAhVFwIMV3ClHwxDGKnLskgicdhc1RqXxVg"
+    assert last_response.ok?
+  end
 end
```

Travis CI needs Google Drive credentials though.

**Comment by dentarg** [comment]
Partially added in #8

**Comment by dentarg** [comment]
Solved by 6133af51, or rather 6544bcb6b8183bed9b59212f928978009cf4df6b
